### PR TITLE
Phalcon devtools

### PR DESCRIPTION
**Pull request by cyberhck**
docker images for devtools

### CODE REVIEW

**Comment by cyberhck** [comment]
@sergeyklay Please review :thumbsup:

**Comment by cyberhck** [comment]
@sergeyklay it's a working image, with the documentation added, you can safely merge and push this.

File: devtools/7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+
+# Pull base image
+FROM phalconphp/composer:7
+
+MAINTAINER Serghei Iakovlev <serghei@phalconphp.com>
+
+# Set up the application directory
+VOLUME ["/app"]
+RUN apk update && apk add git && composer global require phalcon/devtools \
```
Comment (niden) [inline devtools/7/Dockerfile]: You can change this to :

`apk add --no-cache --update add git && composer global require phalcon/devtools`

or clear the cache in the end

Comment (cyberhck) [inline devtools/7/Dockerfile]: done :)

**Comment by sergeyklay** [comment]
I will keep this open for a day or two to allow other contributors a chance to speak up, and to take a fresh look at it later. But at the moment it seems good to me.

**Comment by duythien** [comment]
It will work fine some command but it will not working for option model 

```
$ phalcon model abc

Phalcon DevTools (3.1.2)

ERROR: could not find driver
```

So I think you should extend base image phalconphp/php or phalconphp/php-apache

**Comment by kideny** [comment]
i use vagrant and phalcon box for develop project。i use docker for production env.  Is the phalcon devtools should be  installed to phalconbox ?

**Comment by cyberhck** [comment]
Since I don't use vagrant I can't tell you for sure that it's installed in box project. But I think it should be already included, if not I think you can add it as well :) @sergeyklay knows better than me

**Comment by Fandekasp** [comment]
> I will keep this open for a day or two

:laughing:

**Comment by sergeyklay** [comment]
I apologize @cyberhck for completely overlooking this. Is still actual this feature request? If so, could you please update readme file and other probable outdated things. Thanks again for providing this PR, and again I am sorry about the delay.

**Comment by cyberhck** [comment]
I don't understand how I missed the notification, sorry, my bad!

Actually I moved completely moved away from PHP and into Golang, I don't really have anything here anymore,

Thanks

**Pull request closed without merging**